examples/story-book/README.md

```diff
@@ -0,0 +1,40 @@
+# Story Book
+
+Story Book is a GPTScript that can generate a story based on a prompt and the number of pages you want the story to be in. It is generated in HTML format and can then be viewed
+by `index.html` which has some JS/CSS to make the story styling consistent and readable.
+
+## Usage Instructions
+
+1. **Clone the `image-generation` tool.**
+
+    > Note: This example assumes `image-generation` is cloned as a sibling of your `gptscript` repo. We'll soon have package management that handles tools installation for you, but until then, you have to manually clone and boostrap tools.
+
+    Navigate to the parent directory of your `gptscript` project, then clone the `image-generation` repository:
+
+    ```shell
+    git clone https://github.com/gptscript-ai/image-generation
+    ```
+
+2. **Install and bootstrap the `image-generation` tool.**
+
+    Navigate to the `image-generation` directory, bootstrap the tool, activate the virtual environment, and then return to your original directory:
+
+    ```shell
+    cd ../image-generation
```

Comment:
Where are you assuming I am when I run this command? The root of the gptscript directory? I just did a git clone, presumably in the dir that is the parent of gptscript. After that, `cd ../image-generation` doesn't make sense

I think you should just drop the cd commands and just tell people where they should be.

Reply:
If you told the user to go to the parent dir of gptscript and then combined the clone with this set of steps, that would make sense. They would all work as a single copy-able block